src/generators/generator.js

```diff
@@ -17,7 +17,7 @@ class Generator {
     this.componentName    = args.componentName;
     this.templatePath     = args.templatePath;
     this.testTemplatePath = args.testTemplatePath;
-
+    this.wrapFilesInFolders = args.wrapFilesInFolders;
     // project wide settings
     this.sourceBase    = args.settings.getSetting('sourceBase');
     this.testBase      = args.settings.getSetting('testBase');
```

Comment:
get the wrap files flag from settings here since its going to be T/F for all generators unless at some point we support people being able to pass a flag to not wrap a generated file.  Seems like an all or nothing proposition to me though.. either you want it that way or you don't